src/manifest.json

```diff
@@ -18,9 +18,6 @@
       "run_at": "document_end"
     }
   ],
-  "permissions": [
-    "https://github.com/*",
-    "https://*.giphy.com/*"
-  ],
+  "permissions": ["https://github.com/*", "https://*.giphy.com/*"],
```

Comment:
Thank you for the quick fix! Just a heads up, Chrome today had disabled the extension for me until I re-accepted the permission for access to giphy.com. IDK why it did this as nothing changed here except whitespace...